cli/src/main/kotlin/commands/ReporterCommand.kt

```diff
@@ -29,6 +29,7 @@ import com.beust.jcommander.Parameters
 
 import com.here.ort.CommandWithHelp
 import com.here.ort.model.OrtResult
+import com.here.ort.model.config.CopyrightGarbage
```

Comment:
The commit message still talks about "blacklisting".

Reply:
I kept this because for me it seems clear from the context of just the commit message what it is talking about. However let me change the message again.